### PR TITLE
Completed statistics Screen

### PR DESCRIPTION
**Pull request by mail2bishnoi**
Hi Anton,

Something went wrong with the old statistics branch, please review this for the reverted changes.

### CODE REVIEW

**Review by tananaev — CHANGES_REQUESTED** [review]
I think there is some error again.

**Comment by mail2bishnoi** [comment]
Sorry, Anton, I could not find an error.

**Comment by tananaev** [comment]
You did the opposite of what needs to be done. You removed statistics page, but left filter changes. Have you looked at the diff?

**Comment by mail2bishnoi** [comment]
Hi Anton,

I have added the StatisticsPage.js. I think now its complete.

**Comment by tananaev** [comment]
Why are there still changes in `ReportFilter.js`?

**Comment by tananaev** [comment]
Please take your time and carefully look at the pull request before asking for another review.

**Comment by mail2bishnoi** [comment]
Hi Anton,

I think there is something misunderstood.
When I completely separated the StatisticsPage then you asked me to revert the changes. I understood that revert the changes to use ReporFilter.js with showDevices configuration.
Therefore now ReportFilter.js has changes.

**Comment by mail2bishnoi** [comment]
My questions

1. Should I sperate the StatisticsPage completely means keeping date filter in the same file?
2.  Should I use ReportFilter.js with showDevices param?

**Comment by tananaev** [comment]
When I asked you to revert, you had those changes. So by revert I meant remove the changes.

**Comment by mail2bishnoi** [comment]
Hi Anton,

Again reverted the changes.  Removed configuration from ReportFilter.js. And StatisticsPage has its own period filter.

Sorry, this small PR took longer than expected time due to my misunderstanding. Hope everything is ok now.

File: modern/src/admin/StatisticsPage.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        <Select value={period} onChange={(e) => setPeriod(e.target.value)}>
+          <MenuItem value="today">{t('reportToday')}</MenuItem>
+          <MenuItem value="yesterday">{t('reportYesterday')}</MenuItem>
+          <MenuItem value="thisWeek">{t('reportThisWeek')}</MenuItem>
+          <MenuItem value="previousWeek">{t('reportPreviousWeek')}</MenuItem>
+          <MenuItem value="thisMonth">{t('reportThisMonth')}</MenuItem>
+          <MenuItem value="previousMonth">{t('reportPreviousMonth')}</MenuItem>
+          <MenuItem value="custom">{t('reportCustom')}</MenuItem>
+        </Select>
+    </FormControl>
```
Comment (tananaev) [inline modern/src/admin/StatisticsPage.js:68]: There are some problems with indentation. Please make sure formatting everywhere is correct.

File: modern/src/reports/ReportFilter.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ const ReportFilter = ({ children, handleSubmit, showOnly }) => {
       )}
       {children}
       <FormControl margin="normal" fullWidth>
-        <ButtonGroup color="primary" orientation="vertical" disabled={!deviceId}>
+        <ButtonGroup color="primary" orientation="vertical" disabled={ !deviceId }>
```
Comment (tananaev) [inline modern/src/reports/ReportFilter.js:102]: Please revert this as well.

**Comment by mail2bishnoi** [comment]
Done.

**Comment by tananaev** [comment]
Merged, thank you.

**Pull request merged**